src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/entities/ReportedAdjudication.kt

```diff
@@ -123,7 +123,12 @@ data class ReportedAdjudication(
 
   fun getPunishments() = this.punishments.filter { it.deleted != true }
 
-  fun addPunishment(punishment: Punishment) = this.punishments.add(punishment)
+  fun addPunishment(punishment: Punishment) {
+    if (punishment.sanctionSeq == null) {
```

Comment:
the reason why some sequences are null,  is that we dont capture them from nomis.  The only two we maintain are the damages owed and caution, so its possible this sequence is not  the same as nomis when generated this way?